### PR TITLE
refactored Context, added context for Connections

### PR DESCRIPTION
**Pull request by PavelVesnin**

### CODE REVIEW

File: apps/ehealth/lib/ehealth/web/router.ex [hunk 1]
```diff
@@ -42,8 +42,17 @@ defmodule EHealthWeb.Router do
     plug(:put_is_active_into_params)
   end
 
+  pipeline :connection_context do
+    params = [
```
Comment (AlexKovalevych) [inline apps/ehealth/lib/ehealth/web/router.ex]: why do you need params here?

Comment (PavelVesnin) [reply apps/ehealth/lib/ehealth/web/router.ex]: because of formatter on Travis for Elixir 1.6.6.

Comment (AlexKovalevych) [reply apps/ehealth/lib/ehealth/web/router.ex]:
```
    plug(
      :process_client_context_for_list,
      context_param_name: "allowed_client_id",
      context_params_resolver: :get_connection_context_params
    )
```

**Comment by PavelVesnin** [comment]
@AlexKovalevych please look at next commit:
https://github.com/edenlabllc/ehealth.api/pull/3120/commits/71a44be5940e3c34fa8d261e50a2a7338c8c4bd9

**Pull request merged**